db/migrate/20160101021129_populate_languages.rb

```diff
@@ -0,0 +1,10 @@
+class PopulateLanguages < ActiveRecord::Migration
+  def up
+    Language.create({id:1,name:'English',code:'EN'})
+    Language.create({id:2,name:'Español',code:'ES'})
```

Comment:
That would explain some oddness that I was seeing when running my unit tests earlier today!  Because the previous rails work that I've done was not test-driven.  Thanks for the pointer in the right direction and I'll fix that!